build/dependency-info.sh

```diff
@@ -23,14 +23,18 @@
 #   SPARK_VER - The version of spark
 
 # Parse cudf and spark dependency versions
+set -e
 
 CUDF_VER=$1
 CUDA_CLASSIFIER=$2
 SERVER_ID=snapshots
-${WORKSPACE}/jenkins/printJarVersion.sh "cudf_version" "${HOME}/.m2/repository/ai/rapids/cudf/${CUDF_VER}" "cudf-${CUDF_VER}" "-${CUDA_CLASSIFIER}.jar" $SERVER_ID
+# set defualt values for 'M2DIR' & 'WORKSPACE' so that shims can get the correct cudf/spark dependnecy
+M2DIR=${M2DIR:-"$HOME/.m2/repository"}
+WORKSPACE=${WORKSPACE:-"../.."}
```

Comment:
How do you know what the current working directory is?  This could be called from any directory. You should get the path of this script and then set a path relative to it.  Even better if you can make it an absolute path.

Reply:
@revans2  Good suggest, thanks a lot!